Make.versions

```diff
@@ -43,5 +43,5 @@
 #   line changed in git).
 #
 
-IOS_PACKAGE_VERSION=13.18.0.$(IOS_COMMIT_DISTANCE)
-MAC_PACKAGE_VERSION=6.18.0.$(MAC_COMMIT_DISTANCE)
+IOS_PACKAGE_VERSION=13.18.1.$(IOS_COMMIT_DISTANCE)
```

Comment:
@spouliot should we go `13.18.1` or just bump to `13.20.0` and bump master to `.21`?

Reply:
hard to be sure before _all the things_ gets released
Since d16-6 is still in preview then it makes sense to bump the 3rd digit (as you did)
It won't cause issues for release notes (like the d16-5 backport)